.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: ZGrab2 build action
+on: [push, pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Set up Docker for integration tests
+        uses: docker-practice/actions-setup-docker@master
+
+      - name: Check out source
+        uses: actions/checkout@v2
+
+      - name: Install goimports
+        run: sudo apt install golang-golang-x-tools
```

Comment:
Why do we need `goimports`?

Reply:
Apparently goimports didn't need to be installed separately in TravisCI (based on the .travis.yml), but it does in GitHub Actions. It is used in the Makefile: https://github.com/zmap/zgrab2/blob/5e9507cacf0a7f5cd57aa17a48b29ce6cde76de9/Makefile#L21

I didn't try to understand why the Makefile was the way it was, so it's possible that the Makefile needs cleaning up as well.

Reply:
Let's skip checking formatting in CI for now.